config/gateway.go

```diff
@@ -38,10 +38,6 @@ type Gateway struct {
 	// should be redirected.
 	RootRedirect string
 
-	// DEPRECATED: Enables legacy PUT/POST request handling.
-	// Modern replacement tracked in https://github.com/ipfs/specs/issues/375
-	Writable Flag `json:",omitempty"`
-
```

Comment:
People don't read docs, and for sure they dont read comments in the code.
There is no way someone already using this knew we deprecated it at some point. 
Deprecation notice was added only so the NEW people will not start using it for NEW things.

Removing this without any heads-up will break legacy apps, annoy users by wasting their time trying to figure out why things stopped working, and we also lose oportunity to point people at https://github.com/ipfs/specs/issues/375. 

Can we restore this and add error message that points people at https://github.com/ipfs/specs/issues/375, like we do in config.md already? 

Please follow the convention we successfully applied to all the previous deprecations and keep the field in the config, and do `logFatal` in `daemon.go`  with user-friendly message:

https://github.com/ipfs/kubo/blob/1457b4fd4abff0bdcdccbacc26934cb7fa8e8a43/cmd/ipfs/daemon.go#L869-L870

https://github.com/ipfs/kubo/blob/1457b4fd4abff0bdcdccbacc26934cb7fa8e8a43/cmd/ipfs/daemon.go#L620-L622

Reply:
Thanks! I know we printed warning, but there is a big chunk of users who never see these warnings because they never inspect  logs of instance running in Docker etc. `logFatalf` route is the safest, because node refuses to start, so users is forced to check logs, and there they will see actionable error.